### PR TITLE
Trading view integration

### PR DESCRIPTION
**Pull request by haneefghanim**
Integrated trading view chart into the Asgardex application.

### CODE REVIEW

**Comment by haneefghanim** [comment]
@philipstanislaus Looks like the git submodule setup is causing CircleCI to fail here, presumably because CircleCI doesn't have access to the TradingView repo. Any ideas on how to solve?

File: gui/src/organisms/TVChartContainer.tsx [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+    interval: 'D',
+    libraryPath: '/charting_library/',
+    studiesOverrides: {},
+    symbol: 'AAPL',
+    userId: 'public_user_id',
+  }
+
+  private tvWidget: IChartingLibraryWidget | null = null
+
+  public componentDidMount(): void {
```
[the remark this answers is not in the document]
Comment (Jessicawatsonmiller) [reply gui/src/organisms/TVChartContainer.tsx]: @philipstanislaus is it fine for us to merge without fixing this issue?

**Comment by musnit** [comment]
I see, this is a closed source library? I'd say disable CircleCI for now, or just copy the library in instead of using a submodule.

Secondary concern, once we move into full on production with an expected secure, trustable environment, I don't think this library will be too suitable later on.

**Comment by haneefghanim** [comment]
@musnit Yeah it's closed-source. Unfortunately they don't want their code included in a public repo:

https://github.com/tradingview/charting_library/issues/3296

**Comment by musnit** [comment]
i can't access that link

**Comment by haneefghanim** [comment]
@philipstanislaus OK lets go with disabling CircleCI for now. LMK when I'm good to merge this.
 @musnit I'm attaching a screenshot of the contents of that link.

screen shot 2018-12-13 at 5 33 12 pm

**Comment by philipstanislaus** [comment]
@haneefghanim great, I just merged it. 👍

**Pull request merged**